.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,112 @@
+name: Package Bug Report
+description: Something went wrong with the installation of a package? Or did you find an unexpected behavior?
+title: "(packageName) "
+labels: ["01 - _Triaging", "Bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report! Please fill out out the necessary information as complete as you can. Remember, it is better to include too much information, than too little.
+        Additionall, make sure that the issue title starts with the name of the package you was seeing the issue with inside paranthesis.
+
+        **Do NOT report Security related issues here, instead use the Report Abuse on the package page instead. See https://docs.chocolatey.org/en-us/information/security#report-issue**
+  - type: dropdown
+    id: choco-version
+    attributes:
+      label: Chocolatey Version
+      description: |
+        What version of Chocolatey do you have installed on the machine that this Bug occured (Check all versions that apply, if there are any other versions you know of that are affected please add this to the Additional Details box further down)?
+        Do mind that versions older than the latest version that was available 1 year ago will not be supported.
+      multiple: true
+      options:
+        - 0.11.3
+        - 0.11.2
+        - 0.11.1
+        - 0.11.0
+        - 0.10.15
+    validations:
+      required: true
+  - type: dropdown
+    id: choco-license
+    attributes:
+      label: Chocolatey License
+      description: |
+        Select the license that you are using for Chocolatey when you discovered this bug (Open Source users should select None).
+      options:
+        - Professional
```

Comment:
During the selection by the user, it will be shown as `None`, so I don't think we need to have it explicitly.

I forgot to mention it in the PR body, but the issue forms can be tested out on my fork by going here: https://github.com/AdmiringWorm/chocolatey-coreteampackages/issues/new/choose

Reply:
Awesome. Just had a look and yeah, it's already `None` so ignore this one.